### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by rfair404** [Danielewin8/Games-Direct#3]
How are you document your app? A simple readme might do the trick to begin with. If there are any important things to know about how the application functions, make sure that you write those somewhere.

**Comment by Danielewin8** [comment]
Yup I plan to do a readme including most of if not all the info. As i begin making it I'll try to periodically add corresponding stuff info/instructions.
